src/components/BalanceWithFiat.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect, useRef } from 'react'
+import CountUp from 'react-countup'
+import { Text, TextProps } from '@pancakeswap/uikit'
+
+interface BalanceWithFiatProps extends TextProps {
+  cakeValue: number
+  fiatValue: number
+  fiatDecimals?: number
+  currencySymbol?: string
+  isDisabled?: boolean
+  prefix?: string
+  onClick?: (event: React.MouseEvent<HTMLElement>) => void
+}
+
+const BalanceWithFiat: React.FC<BalanceWithFiatProps> = ({
```

Comment:
We could probably avoid creating BalanceWithFiat by passing `display="inline"` to existing Balance (since the all props are forwarded into Text component down the line). Or if it doesn't work you could make a wrapper like this
  ```javascript
  const StyledBalance = styled(Balance)`
    display: inline;
  `
  ```